tox.ini

```diff
@@ -108,30 +92,6 @@ commands =
     pkg: python -m twine check generated/nise/dist/*
     pkg: python -m twine check generated/niswitch/dist/*
     # pkg: check-manifest --ignore tox.ini,tests*,.github,.github/*,CONTRIBUTING.rst,docs,docs/*
-    nidcpower_system_tests: python --version
-    nidcpower_system_tests: python -c "import platform; print(platform.architecture())"
-    nidcpower_system_tests: python -m pytest src/nidcpower/system_tests --json=generated/kibana/nidcpower_system_test_result.json --durations=5
-    nidigital_system_tests: python --version
-    nidigital_system_tests: python -c "import platform; print(platform.architecture())"
-    nidigital_system_tests: python -m pytest src/nidigital/system_tests --json=generated/kibana/nidigital_system_test_result.json --durations=5
-    nidmm_system_tests: python --version
-    nidmm_system_tests: python -c "import platform; print(platform.architecture())"
-    nidmm_system_tests: python -m pytest src/nidmm/system_tests --json=generated/kibana/nidmm_system_test_result.json --durations=5
-    nifgen_system_tests: python --version
-    nifgen_system_tests: python -c "import platform; print(platform.architecture())"
-    nifgen_system_tests: python -m pytest src/nifgen/system_tests --json=generated/kibana/nifgen_system_test_result.json --durations=5
-    niswitch_system_tests: python --version
-    niswitch_system_tests: python -c "import platform; print(platform.architecture())"
-    niswitch_system_tests: python -m pytest src/niswitch/system_tests --json=generated/kibana/niswitch_system_test_result.json --durations=5
-    niscope_system_tests: python --version
-    niscope_system_tests: python -c "import platform; print(platform.architecture())"
-    niscope_system_tests: python -m pytest src/niscope/system_tests --json=generated/kibana/niscope_system_test_result.json --durations=5
-    nimodinst_system_tests: python --version
-    nimodinst_system_tests: python -c "import platform; print(platform.architecture())"
-    nimodinst_system_tests: python -m pytest src/nimodinst/system_tests --json=generated/kibana/nimodinst_system_test_result.json --durations=5
-    nise_system_tests: python --version
-    nise_system_tests: python -c "import platform; print(platform.architecture())"
-    nise_system_tests: python -m pytest src/nise/system_tests --json=generated/kibana/nise_system_test_result.json --durations=5
```

Comment:
Is running system tests completely removed or is it handled in any other file?

I tried running the test by downloading these files and I didn't find any json report.

Reply:
If you use `run_python_system_tests.py` it knows how to run the system tests in their new location. If you are running them using something else, you will need to make changes similar to what was changed in `run_python_system_tests.py`.

Reply:
If`run_python_system_tests.py` isn't being used by the internal NI test suites, then it's dead code and we should consider deleting it.

Need input from @adityabhat7 and @ni-jfitzger

Reply:
tox is not a tool I am very familiar with and I do not really understand the affects of these changes.
However, using the zip from last night's niscope test run, I ran the code that executes our system tests and then ran them, changed the name of or deleted the results file, deleted run_python_system_tests.py and executed again, then compared the results.
I did this with both python 2.7 and python 3.6.
In all instances, a test results file was produced.  In python 3, it was very clear that the only differences in results was duration of each test.  In python 2, results are much harder to compare but the results are there.

**I think it is safe to say that run_python_system_tests.py is dead code.**

Reply:
It is very tricky to use 'run_python_system_tests.py'. Since this file is available at github and we need the same file to download the code. We need to write a script for downloading only run_python_system_tests.py from master and running it.

I didn't find any json report being created. Are we removing it?

Reply:
I added json output.

There are a few ways to handle this:
1. You can get the latest version of the file by downloading [run_python_system_tests.py](https://raw.githubusercontent.com/ni/nimi-python/master/tools/run_python_system_tests.py). There are command line utilities wget and curl, or you can write a very simple script in the language of your choice. Curl is part of Windows. At least Windows 10. This is very simple to do. `curl -o test.py https://raw.githubusercontent.com/ni/nimi-python/master/tools/run_python_system_tests.py`
1. You can keep the logic to download and unzip the latest release artifact in your test suite, and then invoke a script in the tools directory to actually run the system tests. This is similar to what we do for running system tests in the CI. `tools/system_tests.bat` is used by the Jenkins jobs to actually invoke the test. This way these jobs don't need to be updated every time the system tests change.
1. A combination of the first two. Move the logic that downloads and unzips the release artifact into one script. This won't change unless GitHub's rest API changes. Have a second script in tools to invoke the system tests that you run from the unzipped artifact. 

I think the last one is the best from a maintenance standpoint. Also, the script you invoke to run the system tests will match the system tests you are running. I.e. the ones in the release artifact.

Reply:
I went ahead and implemented option 3. There is now a `prepare_python_system_tests.py` that will download the latest release artifact, unzip it, then execute `run_python_system_tests.py` from the downloaded artifact.

Here is a simple batch file that would use it:
``` batch
mkdir %temp%\nimi-python
chdir %temp%\nimi-python
curl -o prepare_python_system_tests.py https://raw.githubusercontent.com/ni/nimi-python/master/tools/prepare_python_system_tests.py
python prepare_python_system_tests.py --driver niscope --python-version py36
```